src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,139 @@
+import {
+  FC, memo, useEffect, useMemo, useState
+} from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { getVisiblePeople } from '../../utils/getVisiblePeople';
+import { Loader } from '../Loader';
+import { PeopleFilters } from '../PeopleFilter';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { getPersonByName } from './helpers';
+
+export const PeoplePage: FC = memo(
+  () => {
+    const [people, setPeople] = useState<Person[]>([]);
+    const [isLoading, setLoading] = useState(false);
+    const [isError, setError] = useState(false);
+    const [searchParams] = useSearchParams();
+
+    const query = searchParams.get('query');
+    const sex = searchParams.get('sex');
+    const sort = searchParams.get('sort');
+    const order = searchParams.get('order');
+    const century = searchParams.getAll('century');
```

Comment:
you could use custom hook which takes searchParams object and returns all these fields in the object